NAMESPACE

```diff
@@ -914,6 +914,7 @@ importFrom(graphics,symbols)
 importFrom(graphics,text)
 importFrom(graphics,xspline)
 importFrom(graphics,xyinch)
+importFrom(lifecycle,deprecated)
```

Comment:
Are we using this?

Will this conflict with our own `deprecated()` function?

Reply:
that came from `usethis::use_lifecycle()`, now removed